packages/flutter/test/widgets/multichild_test.dart

```diff
@@ -345,4 +352,17 @@ void main() {
 
     checkTree(tester, <BoxDecoration>[kBoxDecorationB, kBoxDecorationC]);
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/37136.
```

Comment:
I've noticed this pattern in a number of recent pull requests. Does the style guide specify referencing issue for tests?

Reply:
Did a quick search in the wiki, wasn't able to find anything related. But I vaguely remember being told once to add this line in a PR. Search for `regression test` in the codebase yields 150 matches, so I guess it's probably a undocumented convention?